eng/Dependencies.props

```diff
@@ -71,6 +68,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="System.CommandLine.Experimental" Version="$(SystemCommandlineExperimentalPackageVersion)" />
     <LatestPackageReference Include="System.ComponentModel" Version="$(SystemComponentModelPackageVersion)" />
     <LatestPackageReference Include="System.ComponentModel.Annotations" Version="$(SystemComponentModelAnnotationsPackageVersion)" />
+    <LatestPackageReference Include="System.Diagnostics.DiagnosticSource" Version="$(SystemDiagnosticsDiagnosticSourcePackageVersion)" />
```

Comment:
Out of curiosity, we previously get this transitively thru aspnetcore-tooling?

Reply:
Previously, this was used in aspnetcore-tooling test projects and not needed in aspnetcore. Now that the tests are migrated, we need this reference.